tests/beaker_tests/cisco_hsrp/test_interface_hsrp.rb

```diff
@@ -82,7 +82,12 @@
 def cleanup(agent)
   cmd = 'no feature hsrp'
   test_set(agent, cmd)
-  interface_cleanup(agent, @intf)
+  interfaces = get_current_resource_instances(agent, 'cisco_interface')
+  interfaces.each do |interface|
+    if interface =~ %r{port-channel100}
```

Comment:
could we use

`    if interface =~ %r{#{@intf}} `

If we are only looking for this interface before clearing it down, get rid of the hard coded string which was already defined as @intf and also means it's more generic :)

Reply:
Good spot!